src/models/room.js

```diff
@@ -346,13 +347,30 @@ Room.prototype.userMayUpgradeRoom = function(userId) {
 Room.prototype.getPendingEvents = function() {
     if (this._opts.pendingEventOrdering !== "detached") {
         throw new Error(
-            "Cannot call getPendingEventList with pendingEventOrdering == " +
+            "Cannot call getPendingEvents with pendingEventOrdering == " +
```

Comment:
thank you for fixing this - I spent  far too much time debugging an issue here recently only to discover the function doesn't exist >:(